package.json

```diff
@@ -22,9 +22,10 @@
     "mocha-junit-reporter": "^1.23.3",
     "mochapack": "^1.1.15",
     "webpack-cli": "^3.3.11",
-    "webpack-dev-server": "^3.10.3"
+    "webpack-dev-server": "^3.10.3",
+    "webpack-node-externals": "^1.7.2"
   },
   "scripts": {
-    "test": "mochapack --webpack-config config/webpack/test.js --require test/setup.js test/**/*.spec.js"
+    "test": "mochapack --webpack-config config/webpack/test.js --require spec/javascript/setup.js --colors 'spec/javascript/**/*.spec.js'"
```

Comment:
Without the quotes around the glob pattern, it's possible for some wonky shell interpolation to kick in instead of letting mocha handle it